### PR TITLE
edit readme

### PR DESCRIPTION
**Pull request by almubarok**
### what changed

- update readme, add info how to manage branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,11 @@
 - [x] basic 1
 - [ ] basic 2
 
+
+#### branch
+- new branch`git branch [nama-branch]`
```
Comment (almubarok) [inline README.md:14]: ini kurang spasi nih setelah branch

File: README.md [hunk 2]
```diff
@@ -9,4 +9,11 @@
 - [x] basic 1
 - [ ] basic 2
```
Comment (rzkimdn) [inline README.md:10]: test

File: oop2/Basic3.java [hunk 3]
```diff
@@ -5,4 +5,9 @@ public class Basic3 {
     public String toString() {
         return super.toString();
     }
+
+    @Override
```
Comment (almubarok) [inline oop2/Basic3.java]:
```suggestion
    @Override // untuk override method parent
```

File: README.md (hunk 1 above)
Comment (almubarok) [reply README.md:14]: gapapa gini doang

Comment (almubarok) [reply README.md:14]: yaudah biar aja

**Review by rafiimalfa — APPROVED** [review]

**Pull request merged**